README.md

```diff
@@ -185,3 +186,21 @@ Like `awesome-developer-streams`? Reach out to [@bitandbang](https://twitter.com
 - [YouTube](https://www.youtube.com/c/SerpentAI)
 - [Discord](https://discord.gg/9D5SuxH)
 - [Patreon](https://www.patreon.com/serpent_ai)
+
+### Gary Kramlich
+#### What Grim Streams
+- Pidgin Development, Convey Development, Open Source Development
+- C, Golang, Python, JavaScript, Docker, Containers
```

Comment:
Could you put these onto one line? 🙏

Reply:
sure, just figured it's a list, so... :)